blob/blob.go

```diff
@@ -59,6 +59,12 @@ type Writer struct {
 
 	// These fields exist only when w is not created in the first place when
 	// NewWriter is called.
+	//
```

Comment:
For the bug, I meant that we should document this in the public API. I.e., in NewWriter, which takes a ctx, we should document that the context must remain open until after the returned Writer is closed.